next.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     BASE_URL: process.env.BASE_URL || "https://devme.bettersocial.org",
     DOWNLOAD_BETTERSOCIAL_APP_URL:
       process.env.DOWNLOAD_BETTERSOCIAL_APP_URL ||
-      "https://linkdev.bettersocial.org/download",
+      "https://link.helio.social/download",
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The URL for `DOWNLOAD_BETTERSOCIAL_APP_URL` has been changed. Ensure that the new URL "https://link.helio.social/download" is correct and accessible, as incorrect URLs can lead to broken links in your application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->